### DISCUSSION TITLE
missing photos

### DISCUSSION THREAD

**Discussion by Githuber522** [FossifyOrg/General-Discussion#403] [Q&A]
Hi,

i want to switch from the standard app gallerys to fossify gallery.

I tried fossify but i realized that fossify doesn't list all photos.
So in the standard app gallerys i have 1000 photos and with fossify 900 although I put extra the setting
that  fossify consider my folders.

Does somebody know this problem?

**Comment by Aga-C** [comment]
App doesn't see a specific folder, or files are missing in some folder?

**Comment by Githuber522** [comment]
I have a few gallery apps at the same time installed.
So I don't think that files are missing in some folder.
Otherwise the other standard gallery apps wouldn't find the photos, too.

fossify gallery / settings / in general 
there are 2 points, where i can manage folders.
For me it sounds that i can hide or add folders for fossify gallerie.
Or how should I undertstand these 2 setting options?

**Reply by Aga-C** [reply]
> I have a few gallery apps at the same time installed.
So I don't think that files are missing in some folder.
Otherwise the other standard gallery apps wouldn't find the photos, too.

I meant in Fossify Gallery, not on your phone. Is it a specific folder where Fossify Gallery lists fewer photos? Or Fossify Gallery doesn't see a specific folder?

> there are 2 points, where i can manage folders.
For me it sounds that i can hide or add folders for fossify gallerie.
Or how should I undertstand these 2 setting options?

`Manage included folders` is for adding folders that aren't visible in the app. `Manage excluded folders` is for hiding whole folders that you don't want to see.

**Reply by Githuber522** [reply]
> I meant in Fossify Gallery, not on your phone. Is it a specific folder where Fossify Gallery lists fewer photos? Or Fossify Gallery doesn't see a specific folder?

when i check the number of the albums and their content, it looks like they are equal



> `Manage included folders` is for adding folders that aren't visible in the app. `Manage excluded folders` is for hiding whole folders that you don't want to see.

How can I prove if or which folder are visible or are not visible in the app?

**Reply by Aga-C** [reply]
> How can I prove if or which folder are visible or are not visible in the app?

If you don't know any folder that is missing, probably there aren't any. By default, Fossify Gallery should show all folders with photos on your device that are also available for other gallery apps. Including folders is mainly for cases, when folders are globally set to be hidden (e.g. folders with image caches of messaging apps), but despite that you want them to be included.

**Reply by Githuber522** [reply]
Puuh, i found it.  Crazy, by chance.
There are some video files in a folder in my internal storage.
These video files aren't visible in the fossify gallery.
Not as single , not in an album.
But why? It it a wma file

**Reply by Aga-C** [reply]
WMA? Don't you mean WMV? Fossify Gallery doesn't support WMV format. You can raise a feature request for it.

**Reply by Githuber522** [reply]
yep, wma.
Great, we got it. Very happy to find the cause.
Thank you for the support.
I'll think about it
